pkg/config/config_test.go

```diff
@@ -2972,3 +2972,60 @@ var testGlobalConfig = OperatorConfig{
 		},
 	},
 }
+
+func TestRawConfigPop(t *testing.T) {
+	tests := []struct {
+		name     string
+		config   RawConfig
+		key      string
+		expected string
```

Comment:
the expected value is still a string here, so this test isn't validating the new behavior you added that converts to a bool